<launch>

	<arg name="rviz" default="true" />

	<rosparam command="load" file="$(find rliv_gs)/config/config.yaml" />

	<node pkg="rliv_gs" type="fusion_mapping" name="mapping" output="screen" />

	<group if="$(arg rviz)">
		<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find rliv_gs)/rviz_cfg/mapping.rviz" />
	</group>
	
	<node pkg="image_transport" type="republish" name="republish" args="compressed in:=/left_camera/image raw out:=/left_camera/image" output="screen" respawn="true"/>

</launch> 	